cognite_toolkit/_cdf_tk/utils.py

```diff
@@ -533,6 +455,22 @@ def reinitialize_from_auth_variables(self, auth: AuthVariables) -> None:
                 credentials=credentials_provider,
             )
         )
+        self._update_environment_variables()
+        return True
+
+    def _update_environment_variables(self) -> None:
+        """This updates the cache environment variables with the auth
+        variables.
+
+        This is necessary for the .as_string() method to dump correctly.
+        """
+        for field_name in ["cluster", "project", "cdf_url", "scopes", "audience"]:
+            try:
+                field_ = _auth_field_by_name[field_name]
+                env_name = field_.metadata["env_name"]
+            except Exception as e:
+                raise RuntimeError("AuthVariables not created correctly. Contact Support") from e
```

Comment:
Not sure how Support can help here

Reply:
That means the variable was set incorrectly in the AuthVariable. Added a comment.